### PR TITLE
added dark mode

### PR DESCRIPTION
**Pull request by bhlynskyExadel**
switch button on header toggles between dark and light mode

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -7,10 +7,15 @@ import UserPage from './pages/User/UserPage';
...
+import DarkThemeProvider from './DarkThemeProvider';
 
-function App() {
+import { Paper } from '@material-ui/core';
+
+const Router = () => {
+    const classes = useStyles();
     return (
-        <HashRouter>
+        <Paper square className={classes.paper}>
```
Comment (uzhdanava) [inline src/App.js:18]: why paper?

File: src/App.js [hunk 2]
```diff
@@ -37,6 +42,16 @@ function App() {
...
                 </Route>
             </Switch>
+        </Paper>
+    );
+};
+
+function App() {
+    return (
+        <HashRouter>
+            <DarkThemeProvider>
```
Comment (uzhdanava) [inline src/App.js:52]: that not only for dark theme. for any

File: src/theme.js [hunk 3]
```diff
@@ -1,7 +1,26 @@
...
+        h2: {
+            fontSize: '25px',
+        },
+        subtitle1: {
+            fontWeight: 'bold',
+        },
+    },
+    overrides: {
+        MuiFormHelperText: {
+            root: { marginBottom: '-20px' },
```
Comment (uzhdanava) [inline src/theme.js:15]: better not use negative values

File: src/App.js (hunk 1 above)
Comment (bhlynskyExadel) [reply src/App.js:18]: Becouse it works good as a background for dark theme, not sure if there any other components for it.

**Review by uzhdanava — APPROVED** [review]

**Pull request merged**